pkg/containertools/dockerfilegenerator.go

```diff
@@ -0,0 +1,98 @@
+package containertools
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	baseImage = "scratch"
+	defaultBinarySourceImage = "quay.io/kevinrizza/upstream-opm-builder"
```

Comment:
probably should be in operator-framework 😄

Reply:
yeah, this is one of the things that needs to change once this pr is rebased. :)